.eslintrc

```diff
@@ -52,13 +52,15 @@
     "arrow-body-style": 1,
     "import/prefer-default-export": 1,
     "import/extensions": 0,
+    "import/no-extraneous-dependencies": 0,
```

Comment:
We use `"import/no-extraneous-dependencies": ["error", {"devDependencies": true}],` in rhinofront, I think this should match.